### DISCUSSION TITLE
How to change text alignment

### DISCUSSION THREAD

**Discussion by ShwayanHu** [academicpages/academicpages.github.io#2359] [Q&A] [answered]
Currently my word are left-aligned? Could you please tell me how can I change it into both-aligned? Thanks!

**Comment by rjzupkoii** [comment, marked as answer]
For the most part the template doesn't do any additional text alignment, so you need to be slightly handy with HTML and CSS to update everything to be justified text. The basic pattern is like this:

```css
div {
  text-align: justify;
  text-justify: inter-word;
}
```

Which says that when the web browser encounters the `div` tag, it should justify the text within and do so based upon the space between words (see [this link for all of the options](https://www.w3schools.com/cssref/css3_pr_text-justify.php)).

Since there's not really a single place where you can add that and have it apply to everything you have a couple options. The simplest approach is to just wrap all of your posts like this:

```html
<div style="text-align: justify; text-justify: inter-word;">
   Your content here!
</div>
```

Which will justify the text in each post and leave everything else as is.

**Reply by ShwayanHu** [reply]
I haven't code HTML or CSS before, but I tried your solution. It works! Thank you very much!
